agent/azure-monitor-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/PreAggregatedMetricsTest.java

```diff
@@ -0,0 +1,368 @@
+            .put("http.flavor", "2.0")
+            .put("http.server_name", "server")
+            .put("http.status_code", 200)
+            .put("http.response_content_length", 200)
+            .build();
+
+    Context parent =
+        Context.root()
+            .with(
+                Span.wrap(
+                    SpanContext.create(
+                        "ff01020304050600ff0a0b0c0d0e0f00",
+                        "090a0b0c0d0e0f00",
+                        TraceFlags.getSampled(),
+                        TraceState.getDefault())));
+
+    Context context1 = listener.onStart(parent, requestAttributes, nanos(100));
+    listener.onEnd(context1, responseAttributes, nanos(250));
+
+    Collection<MetricData> metricDataCollection = metricReader.collectAllMetrics();
+    for (MetricData metricData : metricDataCollection) {
+      System.out.println("metric: " + metricData);
+    }
+
+    assertThat(metricDataCollection.size()).isEqualTo(1);
+
+    assertThat(metricDataCollection)
+        .satisfiesExactly(
+            metric ->
+                assertThat(metric)
+                    .hasName("http.client.duration")
+                    .hasUnit("ms")
+                    .hasHistogramSatisfying(
+                        histogram ->
+                            histogram.hasPointsSatisfying(
+                                point ->
+                                    point
+                                        .hasSum(150 /* millis */)
+                                        .hasAttributesSatisfying(
+                                            equalTo(SemanticAttributes.NET_PEER_NAME, "localhost"),
+                                            equalTo(SemanticAttributes.NET_PEER_PORT, 1234),
+                                            equalTo(SemanticAttributes.HTTP_METHOD, "GET"),
+                                            equalTo(SemanticAttributes.HTTP_FLAVOR, "2.0"),
+                                            equalTo(SemanticAttributes.HTTP_STATUS_CODE, 200))
+                                        .hasExemplarsSatisfying(
+                                            exemplar ->
+                                                exemplar
+                                                    .hasTraceId("ff01020304050600ff0a0b0c0d0e0f00")
+                                                    .hasSpanId("090a0b0c0d0e0f00")))));
+
+    MetricTelemetryBuilder builder = MetricTelemetryBuilder.create();
+    MetricData metricData = metricDataCollection.iterator().next();
+    MetricDataMapper.updateMetricPointBuilder(
+        builder, metricData, metricData.getData().getPoints().iterator().next(), true, true);
+    TelemetryItem telemetryItem = builder.build();
+    MetricsData metricsData = (MetricsData) telemetryItem.getData().getBaseData();
+
+    assertThat(metricsData.getProperties())
+        .containsExactlyInAnyOrderEntriesOf(generateExpectedDependencyCustomDimensions("http"));
+  }
+
+  @SuppressWarnings("SystemOut")
```

Comment:
```suggestion
```

Reply:
line 195.

Reply:
sorry, I meant that we should remove the System.out.println (and this suppression)

Reply:
i can comment it out, but i would like to keep it so that i can easy check otel pre-agg metric json data when running tests.

Reply:
ok